buildpipeline/pipeline.json

```diff
@@ -150,7 +150,8 @@
         "TreatWarningsAsErrors": "false"
       },
       "BuildParameters": {
-        "PB_ConfigurationGroup": "Release"
+        "PB_ConfigurationGroup": "Release",
+        "PB_BuildArguments": "-stripSymbols"
```

Comment:
Sure. To lay out the sequence:

 * `-stripSymbols` unset: symbols are left in the `so`/`dylib` files. (Linux package maintainers typically need to do stripping during their own process, not during our builds.)

 * `-stripSymbols` set: symbols are stripped out of the `so`/`dylib` files into `so.dbg`/`dylib.dwarf` files. The package build has no knowledge of `-stripSymbols`: it creates symbol packages if `.so.dbg`/`dylib.dwarf` files exist in the file system. (See [src/Native/pkg/dir.targets](https://github.com/dotnet/corefx/blob/9da33835697551cf2b35f6cbf59dd1b988c93bd1/src/Native/pkg/dir.targets).)

(This is the same in https://github.com/dotnet/coreclr/pull/9664.)

Would it help to add purpose/result details to [build-native.sh#L18](https://github.com/dotnet/corefx/blob/9da33835697551cf2b35f6cbf59dd1b988c93bd1/src/Native/build-native.sh#L18)?